integration/test_aggregation_integration.py

```diff
@@ -1363,3 +1363,46 @@ def test_non_existing_key_query_by_key_from_v3io_key_is_list(setup_teardown_test
     controller.emit({'nameeeee': 'katya'}, 'katya')
     controller.terminate()
     controller.await_termination()
+
+
+def test_multiple_keys_int(setup_teardown_test):
```

Comment:
Maybe mention in a comment that this is regression for ML-707.

Reply:
I don't think we need an integration test to test this, a unit test would suffice.